lib/castle-rb/client.rb

```diff
@@ -11,17 +11,30 @@ def initialize(request, response)
     end
 
     def identify(args)
-      @api.request('identify', args)
+      @api.request('identify', args) unless do_not_track?
     end
 
     def authenticate(args)
       @api.request('authenticate', args)
```

Comment:
Yeah, I guess we might need to handle this at the API to allow for proper `authenticate` responses. Do Not Track should return a fake response so it doesn't crash the app.